docs/gh_pages/docs/troubleshooting.md

```diff
@@ -1,6 +1,6 @@
-# Troubleshooting Pebblo
+# Troubleshooting Guide
 
-## Issued found in conda virtual env.
+## A. Issues found while running pebblo server, in conda virtual env.
```

Comment:
nit pick: we are using terminology `pebblo daemon` for pebblo server in this doc, so for consistency lets use `daemon` on this line.